OpenRA.Mods.Common/LoadScreens/BlankLoadScreen.cs

```diff
@@ -77,7 +77,13 @@ public void StartGame(Arguments args)
 			// Load a replay directly
 			if (!string.IsNullOrEmpty(Launch.Replay))
 			{
-				var replayMeta = ReplayMetadata.Read(Launch.Replay);
+				ReplayMetadata replayMeta = null;
+				try
+				{
+					replayMeta = ReplayMetadata.Read(Launch.Replay);
+				}
+				catch { }
+
 				if (ReplayUtils.PromptConfirmReplayCompatibility(replayMeta, Game.LoadShellMap))
```

Comment:
Ok, I guess... I just hope you don't ask for more changes when you see what that involves.